WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -459,6 +461,20 @@ && getIntent().getExtras().getBoolean(ARG_CONTINUE_JETPACK_CONNECT, false)) {
             initSelectedSite();
         }
         mSharedLoginResolver.tryJetpackLogin();
+
+        if (mJetpackBrandingUtils.shouldShowJetpackPoweredBottomSheet()
+            && !mBuildConfigWrapper.isJetpackApp()
+            && isJetpackInstalled()) {
+            mGCMMessageHandler.removeAllNotifications(this);
```

Comment:
Once this disables notifications on `WordPress` app, does it get reflected in the `Notification Settings` view, showing disabled `On` flag?

Reply:
Good point.  this PR is only aimed at testing the disabling works for now.  Not sure, if it really necessary to disable the toggle the switch on Notifications screen given it is going to be removed soon entirely!